src/index.js

```diff
@@ -7,6 +7,8 @@ const debug = require('debug')
 const Big = require('bignumber.js')
 const errcode = require('err-code')
 const migrator = require('ipfs-repo-migrations')
+const filesize = require('filesize')
```

Comment:
`pretty-bytes` is more than half the size of `filesize`.  Any reason not to use it?

Reply:
No reason at all. We're using `pretty-bytes` now.